lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb

```diff
@@ -0,0 +1,17 @@
+module DataUpdateScripts
+  class MoveMascotSettingsBacktoSiteConfig
+    def run
+      return unless Database.table_available?("settings_mascots")
+
+      # Remove obsolete data update script
+      DataUpdateScript.delete_by(file_name: "20210420050256_move_mascot_settings")
+
+      # Migrate back explicitly set values
+      mascot_image_url = SiteConfig.mascot_image_url
```

Comment:
I think this needs to be `Settings::Mascot.image_url` and below needs to be `Settings::Mascot.mascot_user_id`

Reply:
Ugh, that's very likely, a lot of the search/replace was done with `sed` and I must have missed this one when doing a manual review at the end. Thanks!